src/components/rangeFilterInput/index.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { FC } from 'react';
+import {
+  InputGroup,
+  NumberInput,
+  NumberInputField,
+  NumberInputProps,
+} from '@chakra-ui/react';
+
+import Stack from '../stack';
+import InputLeftElement from './InputLeftElement';
+
+type InputElement = 'from' | 'to';
+
+type ChangeCallback = {
+  touched: InputElement;
```

Comment:
Why do we need to know which input is touched?

Reply:
I took that reference from `carforyou`. The initial idea was without it but then I thought it would make sense maybe if you enter just one field for the API so that we have single param that tell us what user input in, but that can be moved with your suggestion below with different interface. Thanks!